scripts/run_registry.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash  
+
+set -e
+
+SCRIPT=$(realpath -s "${0}")
+SCRIPTS_PATH=$(dirname "${SCRIPT}")
+TESTS_PATH=$(realpath -s "${SCRIPTS_PATH}/../tests")
+
+: "${ELMNTL_PREFIX:=}"
+: "${ELMNTL_REGCONF:=${TESTS_PATH}/${ELMNTL_PREFIX}test-registry.yaml}"
+: "${ELMNTL_IPADDR=$(ip -4 addr show eth0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}')}"
```

Comment:
In a similar occasion I used `$(ip addr show $(ip route | awk '/default/ { print $5 }') | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)` so that you don't have to guess the interface name. Not sure if it's more reliable, but that still works for me.

Reply:
Gonna update it with your suggestion ;)